persistence/src/main/scala/com/evolutiongaming/akkaeffect/persistence/Recovering.scala

```diff
@@ -30,10 +30,27 @@ trait Recovering[F[_], S, E, +A] {
     journaller: Journaller[F, E],
     snapshotter: Snapshotter[F, S],
   ): Resource[F, A]
```

Comment:
shouldn't we deprecate the original 
```
def completed(
    seqNr: SeqNr,
    journaller: Journaller[F, E],
    snapshotter: Snapshotter[F, S],
  ): Resource[F, A]
```
?

Reply:
IMO no. Almost all apps will continue using it without any drawbacks

Reply:
then we have to document, when to use each and I am not sure, how to make sure that correct one is used in case, when state-transfer is enabled...

Reply:
well, having two almost similar methods always lead to such questions. I'd better deprecate old one then, thanks